leptonai/cli/pod.py

```diff
@@ -101,6 +102,15 @@ def pod():
     type=str,
     multiple=True,
 )
+@click.option(
+    "--log-collection",
+    "-lg",
+    type=bool,
```

Comment:
what's the default value if user don't specify this flag? (is it None or False?)

Reply:
If the user does not specify this flag, log_collection will default to None. 
Consequently, the lepton_deployment_spec.log field will also be None, 

**and the deployment will rely on the workspace's default configuration (backend operation).**